### ISSUE TITLE
✅ 🐞 Editor loads slowly and throws console errors when trying to edit the Header template in Elementor v3.23.0-beta1

### ISSUE THREAD

**Issue by DanielCyran** [elementor/elementor#27814]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

In Elementor version 3.23.0 beta, I encounter a console error and slow loading of the page editing panel. Sometimes it works better, sometimes worse. This issue does not occur in version 3.22.

```
load-scripts.php?c=1&load%5Bchunk_0%5D=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,wp-polyfill-inert,regenerator-runti&load%5Bchunk_1%5D=me,wp-polyfill,wp-hooks&ver=6.5.4:7 JQMIGRATE: Migrate is installed, version 3.4.1
jquery-migrate.min.js?ver=3.4.1:2 JQMIGRATE: Migrate is installed, version 3.4.1
frontend.min.js?ver=3.23.0-beta1:2  Uncaught TypeError: e.O is not a function
    at frontend.min.js?ver=3.23.0-beta1:2:40153
    at webpackJsonpCallback (styleguide-app-initiator.min.js?ver=3.23.0-beta1:3:13486)
    at webpackJsonpCallback (webpack.runtime.min.js?ver=3.23.0-beta1:2:4680)
    at frontend.min.js?ver=3.23.0-beta1:2:74
post.php:1 [Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://go.microsoft.com/fwlink/?linkid=2048113
```

![image](https://github.com/elementor/elementor/assets/96839460/5c96b192-b608-4bd4-9cdb-794b461fb656)

### Steps to reproduce

I updated Elementor to version 3.23.0 beta and suddenly when editing the header I got this error in the console

### Expected behavior

please solve this problem

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.5.25
	PHP Version: 8.3.7
	PHP Memory Limit: 8G
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8G
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.4
	Site URL: https://sklepkuchenny.pl
	Home URL: https://sklepkuchenny.pl
	WP Multisite: No
	Max Upload Size: 8 GB
	Memory limit: 5GB
	Max Memory limit: 8G
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 Edg/126.0.0.0

== Active Plugins ==
	Elementor
		Version: 3.23.0-beta1
		Author: Elementor.com

	Elementor Pro
		Version: 3.23.0-beta1
		Author: Elementor.com

	Rank Math SEO with AI SEO Tools
		Version: 1.0.221
		Author: Rank Math

	WooCommerce
		Version: 9.0.1
		Author: Automattic


== Must-Use Plugins ==
	WordPress automation by Installatron
		Version: 
		Author: 


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	woocommerce: Active


== Elementor Experiments ==
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active by default
	Flexbox Container: Active
	Grid Container: Active
	Upgrade Swiper Library: Active
	Nested Elements Performance: Active by default
	Optimized Control Loading: Inactive
	Optimized Markup: Inactive by default
	Plugin Onboarding: Active by default
	Default to New Theme Builder: Active
	Header &amp; Footer: Inactive
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive
	Build with AI: Inactive
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Active
	Element Caching: Inactive
	Link In Bio: Inactive by default
	Floating Buttons: Active
	Elementor Editor Events: Inactive by default
	Display Conditions: Active
	Form Submissions: Active by default
	Menu: Active
	Taxonomy Filter: Active
	Search: Active


== Log ==
	
Log: showing 20 of 352024-06-19 11:00:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0-beta5',
  'to' => '3.22.0',
)]
2024-06-19 11:00:46 [info] elementor-pro::elementor_pro_updater Started 
2024-06-19 11:00:46 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-06-19 11:00:46 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-06-19 11:00:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0-beta5',
  'to' => '3.22.0',
)]
2024-06-19 11:00:47 [info] elementor::elementor_updater Started 
2024-06-19 11:00:47 [info] Elementor/Upgrades - _on_each_version Start  
2024-06-19 11:00:47 [info] Elementor/Upgrades - _on_each_version Finished 
2024-06-19 11:00:47 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.0-beta6',
  'to' => '3.22.1',
)]
2024-06-19 11:00:48 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.0-beta6',
  'to' => '3.22.1',
)]
2024-06-24 17:13:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0',
  'to' => '3.23.0-beta1',
)]
2024-06-24 17:13:52 [info] elementor-pro::elementor_pro_updater Started 
2024-06-24 17:13:52 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-06-24 17:13:52 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-06-24 17:13:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0',
  'to' => '3.23.0-beta1',
)]
2024-06-24 17:13:52 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0',
  'to' => '3.23.0-beta1',
)]
2024-06-24 17:13:56 [info] elementor::elementor_updater Started 
2024-06-24 17:13:56 [info] Elementor/Upgrades - _on_each_version Start  
2024-06-24 17:13:56 [info] Elementor/Upgrades - _on_each_version Finished 
2024-06-24 17:13:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.1',
  'to' => '3.23.0-beta1',
)]

JS: showing 3 of 3JS: 2024-06-14 05:45:48 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view2793&quot;) has already been destroyed and cannot be used. 
JS: 2024-06-24 17:30:26 [error X 6][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.23.0-beta1:3:830461] Cannot read properties of undefined (reading &#039;localized&#039;) 
JS: 2024-06-24 17:44:25 [error X 1][../wp-admin/load-scripts.php?c=1&amp;loadchunk_0=wp-polyfill-inert,regenerator-runtime,wp-polyfill,wp-hooks,jquery-core,jquery-migrate,utils&amp;ver=6.5.4:7:28760] Cannot read properties of undefined (reading &#039;value&#039;) 

PHP: showing 4 of 4PHP: 2024-06-19 11:21:04 [warning X 2250][../wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::271] Attempt to read property &quot;ID&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-24 17:14:08 [notice X 16][../wp-content/plugins/elementor/includes/base/controls-stack.php::229] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-24 17:36:55 [warning X 1][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-06-24 17:36:57 [notice X 1][../wp-content/plugins/elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by john4242** [comment]
I think this might not be just an elementor issue.
I did a brand new install of WordPress, version 6.5.5
I installed the latest Elementor. Then tried the beta (3.23 beta). Then went back to elementor 3.20. All exhibit the same issue: when trying to edit a page with elementor, the editor doesn't load and it shows same/similar error messages as in this issue.

I suspect maybe WordPress has updated something that broke things?

![image](https://github.com/elementor/elementor/assets/21320270/4f598b08-08f6-4c02-a4f4-08ab4305d26f)

**Comment by john4242** [comment]
Same issue when I try Elementor 3.18 on the same fresh install of WordPress 6.5.5

![image](https://github.com/elementor/elementor/assets/21320270/28812039-7737-4fe2-8bd1-bc3c07994e54)

**Comment by kociol21** [comment]
Yup. I don't have beta. I have Elementor 3.22.2 with Pro 3.22.1 and after updating to Wordpress 6.5.5 I have a lot of similarly looking bugs in console when trying to edit with elementor:

![image](https://github.com/elementor/elementor/assets/8710354/8a5224f2-2d9e-4fc7-89f4-0a976ad96106)

**Comment by m1ga** [comment]
as mentioned in the beta thread already this is a duplicate of https://github.com/elementor/elementor/issues/24492 (error in styleguide-app-initiator.min.js). It appeared before the beta already. For more infos please visit the linked issue

**Comment by nicholaszein** [comment]
Hey @DanielCyran!

This issue was resolved in Elementor version 3.23.0-beta2. Thank you for reporting it!

Kind regards
